core/src/stages/fixed_state.cpp

```diff
@@ -35,13 +35,17 @@
 /* Authors: Robert Haschke */
 
 #include <moveit/task_constructor/stages/fixed_state.h>
+#include <moveit/task_constructor/cost_terms.h>
+
 #include <moveit/planning_scene/planning_scene.h>
 
 namespace moveit {
 namespace task_constructor {
 namespace stages {
 
-FixedState::FixedState(const std::string& name) : Generator(name) {}
+FixedState::FixedState(const std::string& name) : Generator(name) {
+	setCostTerm(cost::Constant(0.0));
```

Comment:
I know, you like making everything explicit. But what about moving this default cost-term definition into the Stage base class? This way, we would ensure that each stage, even new, user-defined ones, will have a well-defined cost function.
The same applies, of course, for all other locations, where you just define zero costs.